configs/dyhead/atss_swin-l-p4-w12_fpn_dyhead_mstrain_tta_2x_coco.py

```diff
@@ -0,0 +1,174 @@
+_base_ = '../_base_/default_runtime.py'
```

Comment:
Could be based on `./atss_swin-l-p4-w12_fpn_dyhead_mstrain_2x_coco.py`?

Reply:
good suggestion!